src/server/views/helpers/handlebars.js

```diff
@@ -39,8 +40,8 @@ const helpers = {
     block.push(options.fn(this));
   },
 
-  encodeIdAttr(id) {
-    return id.replace(/:| /g, '');
+  hashIdAttr(id) {
+    return crypto.createHash('md5').update(id).digest('hex');
```

Comment:
Any reason not to use `sha256` instead of `md5`?

Reply:
Not a good enough reason. Let me know Github's suggestion syntax doesn't work for you and I'll actually commit/push the change.

```suggestion
    return crypto.createHash('sha256').update(id).digest('hex');
```